beaconnect-react-app/package.json

```diff
@@ -33,16 +33,18 @@
     ]
   },
   "dependencies": {
+    "@apollo/client": "^3.7.11",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^13.0.0",
     "@testing-library/user-event": "^13.2.1",
     "@types/jest": "^27.0.1",
     "@types/node": "^16.7.13",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
+    "graphql": "^16.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.10.0",
+    "react-router-dom": "6",
```

Comment:
What is the purpose of changing this version

Reply:
It changes automatically when I install Apollo server, maybe due to compatibility issue